source/features/open-issue-to-latest-comment.tsx

```diff
@@ -9,6 +11,13 @@ function init(): void {
 	}
 }
 
+function initDashboard(): void {
+	for (const icon of select.all('.js-recent-activity-container :not(a) > .octicon-comment')) {
```

Comment:
Not just the icon. The count needs to be clickable too

Reply:
done